### PR TITLE
feat(tabler): integration of tabler icon

### PR DESCRIPTION
**Pull request by ardasnturk**
- added `tabler-icons` to assets folder to integrate tabler icon family
- added `createIconSet` in config/icons/tabler-icons.tsx to integrate the tabler icon family into the code
- added `tabler-icons` to `fontConfig` to load fonts at app startup
- added `Tabler` import to `Icon.tsx` as a comment line to be able to use it quickly

SVA-1348

## Test:

To test the tabler-icon family, update the icons in `Icons.tsx` with the following icons.

```
About: (props: IconProps) => <NamedIcon name="menu-2" {...props} />,
AddImage: (props: IconProps) => <SvgIcon xml={addImage} {...props} />,
Albums: (props: IconProps) => <NamedIcon name="album" {...props} />,
ArrowDown: (props: IconProps) => <SvgIcon xml={arrowDown} {...props} />,
ArrowDownCircle: (props: IconProps) => <NamedIcon name="circle-arrow-down" {...props} />,
ArrowLeft: (props: IconProps) => <SvgIcon xml={arrowLeft} {...props} />,
ArrowRight: (props: IconProps) => <SvgIcon xml={arrowRight} {...props} />,
ArrowUp: (props: IconProps) => <SvgIcon xml={arrowUp} {...props} />,
Calendar: (props: IconProps) => <SvgIcon xml={calendar} {...props} />,
CalendarToggle: (props: IconProps) => <SvgIcon xml={calendarToggle} {...props} />,
Camera: (props: IconProps) => <NamedIcon name="camera" {...props} />,
Check: (props: IconProps) => <NamedIcon name="check" {...props} />,
Clock: (props: IconProps) => <SvgIcon xml={clock} {...props} />,
Close: (props: IconProps) => <SvgIcon xml={close} {...props} />,
CloseCircle: (props: IconProps) => <NamedIcon name="circle-x" {...props} />,
CloseCircleOutline: (props: IconProps) => <NamedIcon name="square-letter-x" {...props} />,
Company: (props: IconProps) => <NamedIcon name="briefcase" {...props} />,
ConstructionSite: (props: IconProps) => <SvgIcon xml={constructionSite} {...props} />,
Document: (props: IconProps) => <NamedIcon name="script" {...props} />,
DrawerMenu: (props: IconProps) => <SvgIcon xml={drawerMenu} {...props} />,
EditSetting: (props: IconProps) => <SvgIcon xml={editSetting} {...props} />,
EmptySection: (props: IconProps) => <SvgIcon xml={emptySection} {...props} />,
ExpandMap: (props: IconProps) => <NamedIcon name="maximize" {...props} />,
Filter: (props: IconProps) => <SvgIcon xml={filter} {...props} />,
GPS: (props: IconProps) => <SvgIcon xml={gps} {...props} />,
Home: (props: IconProps) => <SvgIcon xml={home} {...props} />,
HeartEmpty: (props: IconProps) => <SvgIcon xml={heartEmpty} {...props} />,
HeartFilled: (props: IconProps) => <SvgIcon xml={heartFilled} {...props} />,
Info: (props: IconProps) => <SvgIcon xml={info} {...props} />,
Link: (props: IconProps) => <SvgIcon xml={link} {...props} />,
Like: (props: IconProps) => <SvgIcon xml={like} {...props} />,
List: (props: IconProps) => <SvgIcon xml={list} {...props} />,
Location: (props: IconProps) => <SvgIcon xml={location} {...props} />,
Lunch: (props: IconProps) => <SvgIcon xml={lunch} {...props} />,
Lupe: (props: IconProps) => <SvgIcon xml={lupe} {...props} />,
Mail: (props: IconProps) => <SvgIcon xml={mail} {...props} />,
Member: (props: IconProps) => <SvgIcon xml={member} {...props} />,
NamedIcon,
NotVerifiedBadge: (props: IconProps) => <SvgIcon xml={notVerifiedBadge} {...props} />,
Ok: (props: IconProps) => <SvgIcon xml={ok} {...props} />,
OParlCalendar: (props: IconProps) => <SvgIcon xml={oParlCalendar} {...props} />,
OParlOrganizations: (props: IconProps) => <SvgIcon xml={oParlOrganizations} {...props} />,
OParlPeople: (props: IconProps) => <SvgIcon xml={oParlPeople} {...props} />,
OwnLocation: (props: IconProps) => <SvgIcon xml={ownLocation} {...props} />,
Pause: (props: IconProps) => <NamedIcon name="player-pause-filled" {...props} />,
Pen: (props: IconProps) => <SvgIcon xml={pen} {...props} />,
Phone: (props: IconProps) => <SvgIcon xml={phone} {...props} />,
Play: (props: IconProps) => <NamedIcon name="player-play-filled" {...props} />,
Plus: (props: IconProps) => <NamedIcon name="plus" {...props} />,
RadioButtonEmpty: (props: IconProps) => <NamedIcon name="circle" {...props} />,
RadioButtonFilled: (props: IconProps) => <NamedIcon name="circle-filled" {...props} />,
RoutePlanner: (props: IconProps) => <SvgIcon xml={routePlanner} {...props} />,
Service: (props: IconProps) => <SvgIcon xml={service} {...props} />,
Settings: (props: IconProps) => <NamedIcon name="settings" {...props} />,
Share: (props: IconProps) =>
  device.platform === 'ios' ? (
    <NamedIcon name="share" {...props} />
  ) : (
    <SvgIcon xml={share} {...props} />
  ),
Send: (props: IconProps) => <SvgIcon xml={send} {...props} />,
SueBroom: (props: IconProps) => <SvgIcon xml={sueBroom} {...props} />,
SueCheckSmall: (props: IconProps) => <SvgIcon xml={sueCheckSmall} {...props} />,
SueClock: (props: IconProps) => <SvgIcon xml={sueClock} {...props} />,
SueClockSmall: (props: IconProps) => <SvgIcon xml={sueClockSmall} {...props} />,
SueEye: (props: IconProps) => <SvgIcon xml={sueEye} {...props} />,
SueEyeSmall: (props: IconProps) => <SvgIcon xml={sueEyeSmall} {...props} />,
SueFace: (props: IconProps) => <SvgIcon xml={sueFace} {...props} />,
SueFaceSmall: (props: IconProps) => <SvgIcon xml={sueFaceSmall} {...props} />,
Surveys: (props: IconProps) => <NamedIcon name="chart-bar" {...props} />,
Trash: (props: IconProps) => <SvgIcon xml={trash} {...props} />,
Unvisible: (props: IconProps) => <SvgIcon xml={unvisible} {...props} />,
Url: (props: IconProps) => <SvgIcon xml={url} {...props} />,
VerifiedBadge: (props: IconProps) => <SvgIcon xml={verifiedBadge} {...props} />,
Visible: (props: IconProps) => <SvgIcon xml={visible} {...props} />,
Volunteer: (props: IconProps) => <NamedIcon name="user-circle" {...props} />,
Voucher: (props: IconProps) => <SvgIcon xml={voucher} {...props} />
```

## Screenshots:

|before|after|before|after|
|--|--|--|--|
Simulator Screenshot - iPhone 15 Pro Max - 2024-08-28 at 12 01 13|Simulator Screenshot - iPhone 15 Pro Max - 2024-08-28 at 11 52 30|Simulator Screenshot - iPhone 15 Pro Max - 2024-08-28 at 12 01 17|Simulator Screenshot - iPhone 15 Pro Max - 2024-08-28 at 11 52 36

### CODE REVIEW

File: src/config/Icon.tsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ import { sueBroom } from '../icons/SUE/broom';
 
 import { colors } from './colors';
 import { device } from './device';
+// import { Tabler } from './icons';
```
Comment (donni106) [inline src/config/Icon.tsx:64]: why this?

File: src/config/fonts.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export const fontConfig = {
+  'tabler-icons': require('../../assets/tabler-icons/2.47.0/tabler-icons.ttf'),
```
Comment (donni106) [inline src/config/fonts.ts:2]: can you put this to the end please, as fonts in first case are text fonts, and now icons in addition

File: src/config/Icon.tsx
Comment (donni106) [inline src/config/Icon.tsx:1]: if we have a new icons folder, shouldn't this Icon file also be moved there? what do you think?

File: src/config/Icon.tsx (hunk 1 above)
Comment (donni106) [reply src/config/Icon.tsx:64]: ahh, the default stays at Ionicons and if other icons are requested like tabler, this needs to be changed in this way. I think this should not be commited.

Comment (donni106) [reply src/config/Icon.tsx:64]: or new idea, let's switch to tabler with 4.x anyways. and for releases that should stay at the old icons, we can keep them when merging, you know?

File: src/config/icons/tabler-icons.tsx
Comment (donni106) [inline src/config/icons/tabler-icons.tsx:1]: let's call the file Tabler as the exported const is named?

File: src/config/Icon.tsx (hunk 1 above)
Comment (ardasnturk) [reply src/config/Icon.tsx:64]: I think it makes a lot of sense. Let's use tabs as standard from now on.  👍

File: src/config/icons/tabler-icons.tsx
Comment (ardasnturk) [reply src/config/icons/tabler-icons.tsx:1]: yes 🙌

File: src/config/Icon.tsx
Comment (ardasnturk) [reply src/config/Icon.tsx:1]: yes it makes sense to have it all in one folder.

**Pull request merged**